README.md

```diff
@@ -15,6 +15,13 @@ The specification contains many examples to start learning Flux.
 
 Flux is currently available in InfluxDB 1.7 and 2.0 or through the REPL that can be compiled from this repository.
 
+To build flux, first install the `pkg-config` utility, and
+ensure the GNU `pkg-config` utility is also installed.
```

Comment:
For markdown files, try to have one sentence per line. It makes it easier to review changes because the diff becomes line based and they all format the same.

Reply:
My editor probably did that.